jdaviz/core/loaders/importers/spectrum_list/test_spectrum_list.py

```diff
@@ -0,0 +1,278 @@
+import numpy as np
+import pytest
+from astropy import units as u
+from astropy.utils.masked import Masked
+from astropy.nddata import StdDevUncertainty
+
+from specutils import Spectrum, SpectrumList, SpectrumCollection
+
+from jdaviz.core.loaders.importers.spectrum_list.spectrum_list import (
+    SpectrumListImporter,
+    # combine_lists_to_1d_spectrum
+)
+
+from jdaviz.core.registries import loader_importer_registry
+
+@pytest.fixture
```

Comment:
Is is worth putting these fixtures up in `conftest.py` in case they'll be useful for other tests?